lib/commands/command-help.bash

```diff
@@ -14,20 +14,17 @@ EOF
 }
 
 asdf_extension_cmds() {
-  local plugins_path ext_cmd_path ext_cmds plugin
+  local plugins_path plugin_path ext_cmd_path ext_cmds plugin
   plugins_path="$(get_plugin_path)"
-  # use find instead of ls -1
-  # shellcheck disable=SC2012
-  for plugin in $(ls -1 "$plugins_path" 2>/dev/null | sed "s#^$plugins_path/##"); do
-    ext_cmd_path="$plugins_path/$plugin/lib/commands"
-    ext_cmds=$(
-      ls -1 "$ext_cmd_path"/command*.bash 2>/dev/null |
-        sed "s#$ext_cmd_path/##"
-    )
```

Comment:
Same thing here. Was able to replace a `ls` and `sed` pipeline with just `basename`.